lib/compiler/src/translator/module.rs

```diff
@@ -97,9 +97,7 @@ pub fn translate_module<'data>(
                 unimplemented!("module linking not implemented. It will only be implemented if/when browsers support it")
             }
 
-            Payload::TagSection(_) => {
-                unimplemented!("exception handling not implemented yet")
-            }
+            Payload::TagSection(t) => parse_tag_section(t, environ)?,
```

Comment:
Since we are enabling this in the parser now, we'll need to make sure that the cranelift compiler fails with an exception/Error instead of panicking